### PR TITLE
Tweaks for making first release

### PR DESCRIPTION
**Pull request by jfy133**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
-[![R-CMD-check](https://github.com/sidora-tools/sidora.core/actions/workflows/check-release.yaml/badge.svg)](https://github.com/sidora-tools/sidora.core/actions/workflows/check-release.yaml)
-[![Coverage Status](https://img.shields.io/codecov/c/github/sidora-tools/sidora.core/master.svg)](https://codecov.io/github/sidora-tools/sidora.core?branch=master)
+[![R-CMD-check](https://github.com/sidora-tools/sidora.core/actions/workflows/check-release.yaml/badge.svg?style=for-the-badge&logo=appveyor)](https://github.com/sidora-tools/sidora.core/actions/workflows/check-release.yaml)
+[![Coverage Status](https://img.shields.io/codecov/c/github/sidora-tools/sidora.core/master.svg?style=for-the-badge&logo=appveyor)](https://codecov.io/github/sidora-tools/sidora.core?branch=master)
+[![Latest release](https://img.shields.io/badge/Version-1.0.0-blue)
```
Comment (nevrome) [inline README.md:3]: There is a special shields badge for the current R package version: GitHub R package version

File: NEW.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# sidora 1.0.0 - Mnemosyne
```
Comment (nevrome) [inline NEW.md:1]: If we call it NEWS.md we don't have to ignore it in the .Rbuildignore, I think. 

e.g. https://github.com/poseidon-framework/poseidonR/blob/master/NEWS.md

File: README.md (hunk 1 above)
Comment (nevrome) [reply README.md:3]:
```
![GitHub R package version](https://img.shields.io/github/r-package/v/sidora-tools/sidora.core)
```

Comment (jfy133) [reply README.md:3]: I thought that referred to the version of R the package supports?

File: NEW.md (hunk 2 above)
Comment (jfy133) [reply NEW.md:1]: image
It's not in there already?

File: README.md (hunk 1 above)
Comment (jfy133) [reply README.md:3]: At least that's how I would understand it...

Comment (nevrome) [reply README.md:3]: Nope - it magics the package version out of the DESCRIPTION file

File: NEW.md (hunk 2 above)
Comment (nevrome) [reply NEW.md:1]: Exactly. We currently get this NOTE in `devtools::check()`:

```
N  checking top-level files
   Non-standard file/directory found at top level:
     ‘NEW.md’
```

To make this disappear we can either add `NEW.md` to the buildignore file OR rename the file to NEWS.md (afaik). The latter is more standard.

Comment (jfy133) [reply NEW.md:1]: Oh sorry completely misread . didn't realise the typo in NEWS.md :laugh:

**Pull request merged**